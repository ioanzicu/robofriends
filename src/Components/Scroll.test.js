import React from 'react'
import { shallow } from 'enzyme'
import Scroll from './Scroll'

it('expect to render Scroll component', () => {
  const scrollComponent = shallow(<Scroll />)
  expect(scrollComponent.debug()).toMatchSnapshot()
})
