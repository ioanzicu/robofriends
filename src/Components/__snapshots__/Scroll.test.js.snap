// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Card component 1`] = `"<div style={{...}} />"`;

exports[`expect to render Scroll component 1`] = `"<div style={{...}} />"`;
